lib/slime.ex

```diff
@@ -68,11 +72,15 @@ defmodule Slime do
       ...> end
       iex> Sample.sample(1, 2)
       "3"
+
+      Note: A HEEx-aware version of function_from_string/5 was not included because it would require importing
+      Phoenix.LiveView.HTMLEngine, creating a dependency on Phoenix.
   """
   defmacro function_from_string(kind, name, source, args \\ [], opts \\ []) do
     quote bind_quoted: binding() do
       require EEx
-      eex = source |> Renderer.precompile()
+
+      eex = Renderer.precompile(source)
```

Comment:
```suggestion
      eex = source |> Renderer.precompile()
```

Removing a formatting change to simplify the diff.